_posts/0000-01-02-c-thompson-aims.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![softy](https://user-images.githubusercontent.com/65258922/81757194-6463f600-9501-11ea-8b9b-93890132e5f6.jpg)

**Softy to ID**